modules/countdown.py

```diff
@@ -36,7 +36,8 @@ def response(self, query, message):
         remaining = self.time(event)
         if remaining is None:
             return "There are no events scheduled."
-        return "There are %d weeks, %d days, %d hours, %d minutes, and %d seconds left until %s." % remaining
+        plurality = tuple(["is" if remaining[0] == 1 else "are"]) + tuple("" if num == 1 else "s" for num in remaining[:5])
+        return "There {0} {6} week{1}, {7} day{2}, {8} hour{3}, {9} minute{4}, and {10} second{5} left until {11}.".format(*(plurality + remaining))
```

Comment:
I think you'd say are in any case, since there are multiple units you're listing.

Reply:
Not a big deal though